### DISCUSSION TITLE
Should I upgrade to 8GB rPi?

### DISCUSSION THREAD

**Discussion by miaszh** [tesla-android/issue-tracker#294] [Q&A]
I have everything running on 4gb rpi since few months mainly the Carplay using the carlinkit module..
Now with the option/need to deal with the android system, should I upgrade to a 8GB rPi?

**Comment by mikegapinski** [comment]
No need. The devices I sell are also 4GB

**Reply by miaszh** [reply]
But wouldn't it run faster and smoother on 8GB?

**Reply by mikegapinski** [reply]
you will only notice something in heavy multitasking, not worth the premium
